### PR TITLE
Add Regal, the Rego linter

### PR DESCRIPTION
**Pull request by anderseknert**
New linter for Open Policy Agent's Rego language.

* [x] I have not changed the `README.md` directly, but it was tempting.

### CODE REVIEW

File: data/tools/regal.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+  - rego
+license: Apache License 2.0
+types:
+  - cli
+source: 'https://github.com/styrainc/regal'
+homepage: 'https://github.com/styrainc/regal'
+description: >-
+  Regal is a linter for the policy language Rego. Regal aims to catch bugs and mistakes
+  in policy code, while at the same time help people learn the language, best practices
```
Comment (mre) [inline data/tools/regal.yml:13]:
```suggestion
  in policy code, while at the same time helping people learn the language, best practices
```

**Review by mre — CHANGES_REQUESTED** [review]
Just one typo. Apart from that looks good. Usually we require at least two tools per tag, but since this is a relatively new language, it's fine. Thanks for adding the additional resources.

File: data/tools/regal.yml (hunk 1 above)
Comment (anderseknert) [reply data/tools/regal.yml]: Ah, good catch! Fixed. I think I rewrote that sentence but missed that. I don't think there are any other static analysis tools for Rego at this point in time, but if I'm made aware of one later, I'll make sure to add it, or ask the author to do so. Thanks Matthias!

**Comment by mre** [comment]
Awesome PR and tool. Thanks Anders. 🚀

**Pull request merged**